### ISSUE TITLE
Seek a few seconds 

### ISSUE THREAD

**Issue by missyraja** [OxygenCobalt/Auxio#281]
#### Describe the feature you want to implement:

I would like to have either a 10s or 15s skip forward or backward of the audio track by pressing twice on the left and right side of the album art

#### Is your feature request related to a problem? Please describe:
 The seekbar does a good job of moving between the audio track but a more granular help would be appreciated 

#### Do other music players handle this? If so, how?

YouTube music and other music streaming services have implemented them. 

#### Why do you think this will improve everyone's usage of Auxio?

It will enhance the usability of auxio and you don't need to add extra ui to the app. The functions can be just a hidden double click on the album art

#### Due Diligence:
- [ ] I have read the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).
- [ ] I have read the [Accepted Additions and Requests](https://github.com/OxygenCobalt/Auxio/blob/dev/info/ADDITIONS.md) document.
- [ ] I have checked for this feature in the [FAQ](https://github.com/OxygenCobalt/Auxio/blob/dev/info/FAQ.md).

**Comment by OxygenCobalt** [comment]
I was thinking of doing something like this, but the UI integration is somewhat hard. Adding more buttons is dumb, and while the album art idea kind-of works, it conflicts with #94.

I could see accepting this by overloading the skip buttons with this functionality. If you long-held on skip next for example, it would begin skipping in +5 second intervals until you stopped pressing it. This is a good compromise I think.

**Comment by missyraja** [comment]
YouTube music has both swipe left or right to skip between tracks and also double clicking left or right on the album art for seeking 10s... May be try out the YouTube music app to see how the functionality works and you could think of a way to implement it in your own way in auxio...

 I personally think both could be implemented as how yt music does as it already feels natural and clutter free..

**Comment by missyraja** [comment]
Screenshots![screen-20221117-095952_exported_7145.jpg](https://user-images.githubusercontent.com/40870763/202358901-26193ff6-bdcb-42ee-b75b-bfe042fb1781.jpg)

![screen-20221117-095952_exported_11302.jpg](https://user-images.githubusercontent.com/40870763/202358918-2db5604c-7898-4cc4-ae85-833416b049db.jpg)

![screen-20221117-095952_exported_13479.jpg](https://user-images.githubusercontent.com/40870763/202358929-a4508c0d-1b50-4c69-b17b-96865529105c.jpg)

**Comment by OxygenCobalt** [comment]
Good point, I was just looking at it from a technical difficulty perspective.

**Comment by missyraja** [comment]
Take your time.. this is my favourite offline music app so far..  I hope it grows more and more...

**Comment by KeronCyst** [comment]
Has this been added yet? This would be so helpful for me!

**Comment by OxygenCobalt** [comment]
No @KeronCyst, I want to do this alongside #94, but there are some very high-priority things I need to do first.

**Comment by ghostelephant** [comment]
I have a thought/suggestion that's related to this topic (but if it would be better as its own separate issue, let me know and I can submit it; or, if you think it's a bad idea, also let me know and I can drop it):

What about adding "seek forward *x* seconds" and "seek backward *x* seconds" as other options for the Custom Notification Action button? It's definitely a less robust solution to time-skipping functionality than the one discussed here, but I'm guessing that would probably also make it easier to implement.

I can't speak for others, but I at least am unlikely to ever use either the Shuffle or Repeat buttons from the notification, but would absolutely use a "back 10 seconds" button there.

**Comment by OxygenCobalt** [comment]
And only having one step button at a time would work well for you @ghostelephant? If that's the case, it's possible, but I'd have to implement it such that the "step" button is in the more logical place (i.e step back is next to skip back, step forward is next to skip forward).

**Comment by kalex3** [comment]
@OxygenCobalt as a sponsor, I want this feature to be prioritized.

**Comment by OxygenCobalt** [comment]
Gonna work on this over the weekend to ship in a patch @kalex3 

My focus remains on addressing bugs but this is small enough for me to implement I think alongaide them

**Comment by OxygenCobalt** [comment]
@kalex3 @akashsriramganapathy @Maha-Rajan 

Just implemented basic stepping functionality, albeit without any visual indicators (I need to find some reference implementations to adapt). Will be doing that also very soon.

**Comment by OxygenCobalt** [comment]
Got a prototype visual indicator going, need to add more and refine though. Want to send a build this week I think @kalex3?

**Comment by OxygenCobalt** [comment]
What do you think of this @kalex3 @akashsriramganapathy @Maha-Rajan?

[Auxio_Canary.zip](https://github.com/user-attachments/files/22062398/Auxio_Canary.zip)

**Comment by kalex3** [comment]
Thanks! This was the only feature I was missing.

**Comment by OxygenCobalt** [comment]
This is done by the way, arriving in v4.1.
